third_party/cmake_modules/FindThrift.cmake

```diff
@@ -99,7 +99,9 @@ else()
   find_package(PkgConfig QUIET)
   pkg_check_modules(THRIFT_PC thrift)
   if(THRIFT_PC_FOUND)
-    set(THRIFT_INCLUDE_DIR "${THRIFT_PC_INCLUDEDIR}")
+    set(THRIFT_INCLUDE_DIR
```

Comment:
@majetideepak To make the Thrift include directory accessible in `velox/connectors/hive/test/CMakeLists.txt`, i need to update here. However, this change seems to have caused a formatting issue [here](https://app.circleci.com/pipelines/github/facebookincubator/velox/42994/workflows/3a19e49e-d908-453b-ac9f-f0fad5ce9d88/jobs/298164). I haven't updated any other code. Do you know why this happened? Thanks.

Reply:
cc @mbasmanova

Reply:
@JkSelf I fixed the need to include thrift here https://github.com/facebookincubator/velox/pull/8484